classes/Cart.php

```diff
@@ -755,6 +755,16 @@ public function getProducts($refresh = false, $id_product = false, $id_country =
                     $reduction_type_row = ['reduction_type' => 0];
                 }
 
+                if (Pack::isPack($product['id_product'])) {
+                    $product['quantity_available'] = Product::getQuantity(
+                        $product['id_product'],
+                        $product['id_product_attribute'],
+                        null,
+                        null,
+                        $product['id_customization'],
```

Comment:
```suggestion
                        $product['id_customization']
```

maybe this is the issue it fails on PHP 7.2?

Reply:
Hi @kpodemski,
in fact it could be.
I run the fix that @Hlavtox proposed which is actually better.
A thousand thanks!